File: Python/day_7_string_range.py
def string_range():
    num=input("Enter a number : ")
    lst=[]
    try:
        num=int(num)
    except:
        return string_range()
    else:
        for i in range(0,num):
            lst.append(str(i))
        lsst=".".join(lst)
        return lsst
output=string_range()
print(output)

File: JavaScript/day_7_string_range.js
const read=require('readline').createInterface({
    input:process.stdin,
    output:process.stdout
});
(function(){
    read.question("Enter the Number : ",number=>{
        let arr=[]
        number=parseInt(number);
        for(let i=0;i<number;i++){
            arr.push(i);
        }
        console.log(arr.join("."));
    })
})();